src/suggestionlistmodel.cpp

```diff
@@ -93,3 +96,9 @@ QModelIndex SuggestionListModel::fetchEndIndex() const
     int trueFetchSize = m_urlList.size() - (m_hasFullText ? 1 : 0);
     return index(trueFetchSize - 1);
 }
+
+void SuggestionListModel::setNoMoreSuggestion(int fetchedSize)
+{
+    int trueFetchSize = fetchedSize - (m_hasFullText ? 1 : 0);
+    m_noMoreSuggestion = trueFetchSize < SuggestionListWorker::getFetchSize();
```

Comment:
This is the answer to my earlier question about `SuggestionListWorker::getFetchSize()` being justified.